components/connector/internal/certificates/service.go

```diff
@@ -44,13 +45,17 @@ func NewCertificateService(
 func (svc *certificateService) SignCSR(encodedCSR []byte, subject CSRSubject) (EncodedCertificateChain, apperrors.AppError) {
 	csr, err := svc.certUtil.LoadCSR(encodedCSR)
 	if err != nil {
+		log.Errorf("Error occurred while loading the CSR with Common Name %s", subject.CommonName)
 		return EncodedCertificateChain{}, err
 	}
+	log.Debugf("Successfully loaded the CSR with Common Name %s", subject.CommonName)
 
 	err = svc.checkCSR(csr, subject)
 	if err != nil {
+		log.Errorf("Error occurred while checking the values of the CSR with Common Name %s", subject.CommonName)
 		return EncodedCertificateChain{}, err
 	}
+	log.Debugf("Successfully checked the values of the CSR with Common Name %s", subject.CommonName)
 
 	return svc.signCSR(csr)
```

Comment:
one more debug log here Successfully signed CSR with common name after signCSR before returning

Reply:
Sorry, didn't understand exactly what the content of the log should be:

`Successfully signed CSR with common name after signCSR before returning` or `Successfully signed CSR with common name after signCSR` and place it before the return statement?

Reply:
2nd (method call, handle error, log success, return)